lib/datadog/core/environment/process.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require_relative 'ext'
+require_relative '../tag_normalizer'
+
+module Datadog
+  module Core
+    module Environment
+      # Retrieves process level information such that it can be attached to various payloads
+      #
+      # @api private
+      module Process
+        extend self
+
+        # This method returns a key/value part of serialized tags in the format of k1:v1,k2:v2,k3:v3
+        # @return [String] comma-separated normalized key:value pairs
+        def serialized
+          return @serialized if defined?(@serialized)
+          tags = []
+          tags << "#{Environment::Ext::TAG_ENTRYPOINT_WORKDIR}:#{TagNormalizer.normalize(entrypoint_workdir, remove_digit_start_char: false)}" if entrypoint_workdir
```

Comment:
Not sure, but when it will be `nil`?

Reply:
That's a good question - I removed after not being able to think of a good reason why it'd be nil: https://github.com/DataDog/dd-trace-rb/pull/5033/commits/f3f14806c1b1838a4b198dd19c4b7ece78801adb